api/src/clojure/org/akvo/flow_api/datastore/form_instance.clj

```diff
@@ -103,9 +104,14 @@
   [_ response-str]
   response-str)
 
+(def ^DateFormat sdf (SimpleDateFormat. "dd-MM-yyyy HH:mm:ss z"))
```

Comment:
`SimpleDateFormat` is not thread safe. The usual way to _solve_ the problem is put it in a ThreadLocal.
https://github.com/akvo/akvo-flow/blob/v1.9.15/GAE/src/org/waterforpeople/mapping/app/web/DataBackoutServlet.java#L71-L76

Can this affect us?